blocks/graph.py

```diff
@@ -204,10 +204,15 @@ def replace(self, replacements):
         # Sort `replacements` in topological order
         for node in apply_nodes_sorted:
             for input_ in node.inputs:
-                if input_ not in replacements:
-                    continue
-                replacement_keys_cur.append(input_)
-                replacement_vals_cur.append(replacements[input_])
+                if input_ in replacements:
```

Comment:
It made me panic for two days, because after I started to use the aforementioned monitoring applying dropout _caused_  severe overfitting. This sort of makes sense, because division by `1 - dropout_prob` happened twice and the training and validation objective were no longer similar, but quite different.

I will fix it tomorrow, if nobody does it before me.